docs/scripts/webpack.config.babel.js

```diff
@@ -97,6 +96,7 @@ export default {
       ),
       CLIENT: true,
       REDUX_DEVTOOLS: Boolean(process.env.REDUX_DEVTOOLS),
+      BASENAME: JSON.stringify(process.env.BASENAME),
```

Comment:
Since this name is reall generic, I think I will rename this to STATINAMIC_PATHNAME (also basename is not coherent with baseUrl.pathname... Maybe we should replace the baseUrl from configurator for a custom object that will have basename in place of pathname?)